intel-retail/rtsf-at-checkout-reference-design
# feat(device_mqtt, device_rest): migrate device-list files for device-…

…mqtt and device-rest to EdgeX v3.0 yaml

## PR Checklist



## What are you changing?


- Updates device-mqtt and device-rest device files to yaml for EdgeX v3.0

## Issue this PR will close

close: #**72**

closes #72 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable


- Run `make run-edgex run-portainer`
- Navigate to `localhost:9000` and check portainer logs of `device-rest` and `device-mqtt`
- Look for `level=INFO ts=2023-10-03T17:32:39.903218108Z app=device-mqtt source=devices.go:54 msg="Loading pre-defined devices from /res/devices(1 files found)"`
 and `level=INFO ts=2023-10-03T17:32:39.832931714Z app=device-rest source=devices.go:54 msg="Loading pre-defined devices from /res/devices(1 files found)"`
- Close with `make clean` and `docker compose -f loss-detection-app/docker-compose.portainer.yml down`

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/rtsf-at-checkout-reference-design )
